praw/config.py

```diff
@@ -29,6 +32,16 @@ class Config:
         "extended": configparser.ExtendedInterpolation,
     }
 
+    @property
+    def custom(self) -> Dict[str, Any]:
```

Comment:
Why are you removing this feature?

Reply:
Since the custom dict used to be the combination of file and hardcoded config values, and now that those have been seperated, there has to be a new way to get config values.